sunpycube/cube/datacube.py

```diff
@@ -558,14 +559,16 @@ def __init__(self, data_list, meta=None, common_axis=None):
         self.data = data_list
         self.meta = meta
         self.common_axis = common_axis
+        self.shape = (len(data_list), data_list[0].shape[0],
+                      data_list[0].shape[1], data_list[0].shape[2])
```

Comment:
```self.shape``` should be calculated more generically so that it is valid for Cubes of any dimensions, e.g.
```
shape = list(data_list[0].shape)
shape.insert(len(data_list), 0)
self.shape = tuple(shape)
```